app/src/main/java/com/example/keybindhelper/RecyclerViewAdapters/KeybindAdapter.java

```diff
@@ -215,17 +206,17 @@ private void showContextMenu(Keybind k, View view) {
             d.cancel();
             ArrowProvider ap=new ArrowProvider(context);
             ap.directionClicked=isUp -> {
-                int indx=k.group.keybinds.indexOf(k);
+                int index=k.group.keybinds.indexOf(k);
                 if(isUp){
-                    if(indx>0) {
+                    if(index>0) {
                         k.group.moveKeybindUpDown(k,-1);
-                        notifyItemMoved(indx,indx-1);
+                        notifyItemMoved(index,index-1);
                         updateKeybindsBackground();
                     }
                 }else{
-                    if(indx<k.group.keybinds.size()-1){
+                    if(index<k.group.keybinds.size()-1){
                         k.group.moveKeybindUpDown(k,1);
-                        notifyItemMoved(indx,indx+1);
+                        notifyItemMoved(index,index+1);
```

Comment:
renamed indx to index to better understand code.

Reply:
Its a good idea to spell out variable names, and make them descriptive.  The time you save in pressing a few keys on an abbreviation will quickly be lost as you try to remember what abbreviation you used later.  :)